Update suse-create-upload-vhd.md

mries opened the pull request:
removed end-of- life version and made adjustments to the instruction examples.

mries committed:
Update suse-create-upload-vhd.md

removed end-of- life version and made adjustments to the instruction examples.

mries committed:
Update disk-encryption-sample-scripts.md

updated version number link

mries committed:
Update suse-create-upload-vhd.md

fixed code blocks

mries committed:
Update suse-create-upload-vhd.md

more code blocks

anhvoms commented on articles/virtual-machines/linux/suse-create-upload-vhd.md:
````diff
@@ -221,33 +221,35 @@ As an alternative to building your own VHD, SUSE also publishes BYOS (bring your
-    sudo zypper ar -f https://download.opensuse.org/distribution/15.2/repo/oss openSUSE_15.2_OSS
-    sudo zypper ar -f http://download.opensuse.org/update/15.2 openSUSE_15.2_Updates
-    ```
+```bash
````
Does the repository path need to change here?

mries replied:
yes this was an oversight.  I had the new repo links tested but didn't add them to the document (or, it looks like a copy paste issue) . That was an oversight. They should look like this:


sudo zypper ar -f https://download.opensuse.org/update/openSUSE-stable openSUSE_15.4_Updates
sudo zypper ar -f https://download.opensuse.org/repositories/Cloud:/Tools/15.4 Cloud:Tools_15.4
sudo zypper ar -f https://download.opensuse.org/distribution/openSUSE-stable/repo/oss openSUSE_stable_OSS

I will update in a patch.

mries committed:
Update suse-create-upload-vhd.md

Updated opensuse 15.4 version table and corrected links for adding repositories.

Jak-MS committed:
adding indents and auto-numbering
